app/controllers/rules_controller.rb

```diff
@@ -7,9 +7,14 @@ def index
   end
 
   def show
-    rule = ::Pundit.policy_scope(User.current, ::Rule).friendly.find(
-      params[:id]
-    )
+    rule = if ::UUID.validate(params[:id])
```

Comment:
This was easier and cleaner than 
`.where('rules.slug LIKE ? OR rules.id = ?', params[:id], params[:id])`, as we would have to validate anyway. Postgres seems to error out if you compare a String with a UUID type, which would happen if anyone searched for a ref_id